internal/pkg/source/source.go

```diff
@@ -126,12 +127,23 @@ func sourcesFromBlocks(fn *ssa.Function, conf *config.Config, taggedFields field
 			default:
 				continue
 
-			// local variable or value returned from a call
-			case *ssa.Alloc, *ssa.Call:
-				if isProducedBySanitizer(v.(ssa.Value), conf) {
+			// Do not mark allocation values returned by sanitizers as new sources
```

Comment:
Nit: I think this comment would be more helpful if it were a bit higher level, e.g. `values produced by sanitizers cannot be sources`.